src/wpsmon.c

```diff
@@ -499,10 +519,15 @@ void parse_wps_settings(const u_char *packet, struct pcap_pkthdr *header, char *
 
 					if(wps_active(wps))
 					{
+						char wps_versao[8];
+						snprintf(wps_versao, sizeof(wps_versao), "%d.%d", (wps->version >> 4), (wps->version & 0x0F));
+						if (is_pbc(bssid)) {
+							strcpy(wps_versao, "PBC");
+						}
 						if (show_crack_progress)
-							fprintf(stdout, "%17s  %3d  %.2d  %d.%d  %3s  %8s  %5s  %s\n", bssid, channel, rssi, (wps->version >> 4), (wps->version & 0x0F), lock_display, vendor ? vendor : "        ", crack_progress ? crack_progress : "-", sane_ssid);
+							fprintf(stdout, "%17s  %3d  %.2d  %s  %3s  %8s  %5s  %s\n", bssid, channel, rssi, wps_versao, lock_display, vendor ? vendor : "        ", crack_progress ? crack_progress : "-", sane_ssid);
 						else
-							fprintf(stdout, "%17s  %3d  %.2d  %d.%d  %3s  %8s  %s\n", bssid, channel, rssi, (wps->version >> 4), (wps->version & 0x0F), lock_display, vendor ? vendor : "        ", sane_ssid);
+							fprintf(stdout, "%17s  %3d  %.2d  %s  %3s  %8s  %s\n", bssid, channel, rssi, wps_versao, lock_display, vendor ? vendor : "        ", sane_ssid);
 					}
 					else
 					{
```

Comment:
i think this commit, while adding the PBC info to regular wash output, makes it so it no longer appears in json

Reply:
I disagree. The SEEN_FLAG_COMPLETE flag is reset whenever PBC changes. One of the conditions to print json is SEEN_FLAG_COMPLETE is 0.